test-network-nano-bash/generate_artifacts.sh

```diff
@@ -0,0 +1,25 @@
+#remove existing artifacts
```

Comment:
Not everybody runs in the bash shell, so would add something like this at the top of these scripts... 

```
#!/usr/bin/env bash
set -euo pipefail
```

Reply:
done